packages/web-components/src/components/va-text-input/va-text-input.scss

```diff
@@ -0,0 +1,70 @@
+@use 'uswds-helpers/src/styles/usa-sr-only';
+@use 'usa-label/src/styles/usa-label';
+@use 'usa-input/src/styles/usa-input';
+@use 'usa-error-message/src/styles/usa-error-message';
+
+@import '../../mixins/uswds-error-border.css';
+
+:host {
+  color: var(--color-base);
+  display: block;
+  font-family: var(--font-source-sans);
+}
+
+:host(:focus) {
+  outline: none !important;
+}
+
+:host([uswds]) input:not([disabled]):focus{	
+  outline: 2px solid var(--color-gold-light);	
+  outline-offset: 2px;	
+}
+
+input {
+  box-sizing: border-box;
```

Comment:
This fixes the issue we were seeing with the input height calculating larger than what it should be.

https://github.com/department-of-veterans-affairs/component-library/pull/587#issuecomment-1343386102